pkg/result/processors/nolint.go

```diff
@@ -30,23 +30,27 @@ func (i *ignoredRange) doesMatch(issue *result.Issue) bool {
 		return false
 	}
 
-	// handle possible unused nolint directives
-	// nolintlint generates potential issues for every nolint directive and they are filtered out here
-	if issue.ExpectNoLint {
-		if issue.ExpectedNoLintLinter != "" {
-			return i.matchedIssueFromLinter[issue.ExpectedNoLintLinter]
+	// only allow selective nolinting of nolintlint
```

Comment:
This has been reorganized so that we can do `//nolint:nolintlint` to disable `nolintlint` checking.